src/commands/synthetics/run-test.ts

```diff
@@ -1,42 +1,47 @@
 import chalk from 'chalk'
 import {Command} from 'clipanion'
+import deepExtend from 'deep-extend'
 
-import {parseConfigFile, ProxyConfiguration} from '../../helpers/utils'
+import {parseConfigFile} from '../../helpers/utils'
 import {apiConstructor} from './api'
-import {APIHelper, ConfigOverride, ExecutionRule, LocationsMapping, MainReporter, PollResult, Test} from './interfaces'
+import {APIHelper, CommandConfig, ExecutionRule, LocationsMapping, MainReporter, PollResult, Test} from './interfaces'
 import {DefaultReporter} from './reporters/default'
 import {Tunnel} from './tunnel'
 import {getReporter, getSuites, getTestsToTrigger, hasTestSucceeded, runTests, waitForResults} from './utils'
 
+export const DEFAULT_COMMAND_CONFIG: CommandConfig = {
+  apiKey: '',
+  appKey: '',
+  configPath: 'datadog-ci.json',
+  datadogSite: 'datadoghq.com',
+  fileGlobs: ['{,!(node_modules)/**/}*.synthetics.json'],
+  global: {},
+  pollingTimeout: 2 * 60 * 1000,
+  proxy: {protocol: 'http'},
+  publicIds: [],
```

Comment:
Is `CommandConfig` expected to match all options available through the config file?
In that case, both `publicIds` and `configPath` should not be here.

Reply:
`CommandConfig` should match all options ( `ENV` U `file` U `CLI`)